lib/rules/lint-attribute-indentation.js

```diff
@@ -285,30 +289,68 @@ module.exports = class AttributeSpacing extends Rule {
     let expectedColumnStart = node.loc.start.column + this.config.indentation; //params should be after proper positions from component start node
     let expectedLineStart = node.loc.start.line + 1;
 
-    expectedLineStart = this.iterateParams(node.params, 'positional', expectedLineStart, expectedColumnStart, node);
```

Comment:
These two lines seem to have been deleted, but I would think they need to end up in the else clause of `if (node.type === 'ElementNode') {` right?

Reply:
Yeah, they are in lines 295-296 (below), am I missing something?